chapters/modules/chapter_1/pages/why_cairo.adoc

```diff
@@ -0,0 +1,20 @@
+[id="whyCairo"]
+
+= Why Cairo?
+
+StarkNet's scalability tool, ZK Roll-ups, enable a new paradigm called Provable Computation. In this paradigm, we allow programs to prove they have been executed without having to be re-run. In order to write such provable programs, we need a programming language that can be effective and be easily learnt by developers. 
+
+If you are looking for a cost-effective provable computation programming language, consider Cairo. It is native to Provable Computation and includes low-level access to memory, underlying primitives, and functional-style programming. Listed below, are some of the benefits of Cairo: +
+
+* It is a specially designed, provable language for smart contracts +
+* It is not bounded by the EVM +
+* Its functions are imported rather than inherited +
+* It supports Solidity to Cairo transpiler +
+* It includes full low-level access to underlying primitives +
```

Comment:
That was the goal of Cairo 0 but not Cairo 1 which strives to be strictly a high level language. Which primitives are you referring to?

Reply:
The sentence in my mind was more to convey something like - it has the ability to work with felts. I changed it to reflect the ease of use